.circleci/config.yml

```diff
@@ -26,9 +25,25 @@ jobs:
             - build-v4-
           paths:
             - /tmp/docker
+            - /tmp/images
```

Comment:
Is it possible we'll want to clear the `registry:2` cache independent of the build cache?

Reply:
I think we'll update that image so infrequently that it's not really worth it to make a new cache 🤷‍♂️